stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -933,8 +856,6 @@
         prepare-cycle-length: (var-get pox-prepare-cycle-length),
         first-burnchain-block-height: (var-get first-burnchain-block-height),
         reward-cycle-length: (var-get pox-reward-cycle-length),
-        rejection-fraction: (var-get pox-rejection-fraction),
-        current-rejection-votes: (next-cycle-rejection-votes),
```

Comment:
The removal of these fields will break `/v2/pox` and lead to a runtime panic.  Please fix `stackslib/src/net/api/getpoxinfo.rs`.

Reply:
done thanks to @kantai